src/app/page/Page.tsx

```diff
@@ -31,6 +31,13 @@ const Page = ({ children }: PageProps) => {
             <InternalNavLink key="applications" to="/applications" icon={<IconApplications />}>
               {t('common.sidebar.applications')}
             </InternalNavLink>,
+            <InternalNavLink
+              key="winterStorageApplications"
+              to="/winter-storage-applications"
+              icon={<IconApplications />}
+            >
+              {t('common.sidebar.winterStorageApplications')}
+            </InternalNavLink>,
```

Comment:
One design shows the applications as a category menu. The `Expandable` component we have in `common` is probably borked ⚡, so this could be done in a separate issue
![image](https://user-images.githubusercontent.com/15479131/87672056-052d9680-c77b-11ea-8048-f97c08a2027a.png)

Reply:
Will create a separate issue about this.